modules/configure_user/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "resource_group_id" {
+  description = "ID of the IBM Cloud resource group in which resources are created."
+  type        = string
+}
+
+variable "region" {
+  description = "The location that's used with the IBM Cloud Terraform IBM provider. It's also used during resource creation."
```

Comment:
Can we modify the description a bit without explicitly mentioning about Provider?

Reply:
updated